src/crawlers/statistics.js

```diff
@@ -245,9 +247,9 @@ class Statistics {
             finishedJobs: this.finishedJobs,
             failedJobs: this.failedJobs,
             totalJobDurationMillis: this.totalJobDurationMillis,
-            startedAt: +this.startedAt,
+            startedAt: this.startedAt ? this.startedAt.toISOString() : null,
             // used for adjusting time between runs recreating from state
-            persistedAt: Date.now(),
+            persistedAt: new Date().toISOString(),
```

Comment:
JSON.stringify automatically converts Dates to ISOStrings. Is this needed because we're overriding `toJSON`?

Reply:
Also, `JSON.stringify(stats)` does not necessarily mean that the stats were persisted. It just means that something called `JSON.stringify`. The `persistedAt` should only be updated when the stats are actually persisted to KVS.

Reply:
it's about consistency, toJSON being called manually, like in tests, will return a iso string instead of a date instance (that would be the same if it was using Date.now(), would be always a number)

Reply:
yeah, I know it doesn't return JSON, but the representation should be the same, if you call it directly or indirectly through JSON.stringify, you should get the same result